src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -131,6 +131,7 @@
 
             <BusinessLookup
               :key="Math.random()"
+              class="mt-5"
```

Comment:
The latest version of the shared component removed the whitespace from the component itself, so that the parent (here) can specify what it needs.